app-2/src/main/java/tech/relaycorp/letro/awala/message/MessageType.kt

```diff
@@ -0,0 +1,25 @@
+package tech.relaycorp.letro.awala.message
+
+sealed class MessageType(val value: String) {
+    object AccountCreationRequest : MessageType("application/vnd.relaycorp.letro.account-creation-request")
+    object AccountCreationCompleted : MessageType("application/vnd.relaycorp.letro.account-creation-completed-tmp")
```

Comment:
@gnarea still tmp?

Reply:
Yes, until we do the VeraId integration and switch to the new server instance.